clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResults.java

```diff
@@ -25,10 +25,10 @@
  * The result of newTopics.
  */
 @InterfaceStability.Unstable
-public class CreateTopicResults {
+public class CreateTopicsResults {
```

Comment:
Thanks for fixing this, I had it in one of my to-do lists somewhere. :) Also, while we are doing this, should it be `CreateTopicsResults` or `CreateTopicsResult`? I prefer the latter a little as it makes it easier to name lists of this class. But maybe the `results` method may look a little out of place (and we may need a new name for it).